src/esp/sensor/CMakeLists.txt

```diff
@@ -1,51 +1,31 @@
 set(sensor_SOURCES
-  PinholeCamera.cpp
-  PinholeCamera.h
-  Sensor.cpp
-  Sensor.h
-  VisualSensor.h
-)
-
+    PinholeCamera.cpp
+    PinholeCamera.h
+    Sensor.cpp
+    Sensor.h
+    VisualSensor.h)
 
 if(BUILD_WITH_CUDA)
-  list(APPEND sensor_SOURCES
-    RedwoodNoiseModel.cpp
-    RedwoodNoiseModel.h
-  )
+    list(
+        APPEND
+        sensor_SOURCES
+        RedwoodNoiseModel.cpp
+        RedwoodNoiseModel.h)
 endif()
 
-
 add_library(sensor STATIC ${sensor_SOURCES})
 
-target_link_libraries(sensor
-  PUBLIC
-    core
-    gfx
-    scene
-)
+target_link_libraries(sensor PUBLIC core gfx scene)
```

Comment:
I'm not a fan of packing these onto one line. It's easier to notice additions/deletions in diffs when each list item is its own line.

Reply:
They only get packed onto one line if they are three or less arguments.

Reply:
Is there some middle ground between "aggressively packing everything with < 4 arguments onto a single line" and "aggressively expanding everything with >= 4 arguments onto several lines"? Something like "always expand `target_link_libraries()`" or "preserve whatever format was there for certain commands"?

Like, in this case I would prefer to have the args separated on multiple lines because it's *very likely* more libs will get added later, however with the `set(thing CACHE BOOL "bla" OFF)` it'll *never happen* that there will be more arguments and spreading it out like that just adds unnecessary congitive load as the `WITH_` options etc are no longer nicely aligned under each other.

Reply:
@mosra There is, is there anything we should expand besides target_link_libraries?

Reply:
I thought about doing something similar to the `set(bla_SOURCES many things here)` commands but most of those lists could be put directly into `add_library()` / `add_executable()` ... and conditional sources later added with `target_sources()`. 

So maybe add `add_executable()` and `target_sources()` to the list as well, even though the latter is probably not used anywhere yet.

Reply:
We can just make a note to add those later once we use those features.